packages/cli/src/lib/bundler/bundle.ts

```diff
@@ -32,6 +32,10 @@ import chalk from 'chalk';
 const WARN_AFTER_BUNDLE_GZIP_SIZE = 512 * 1024;
 const WARN_AFTER_CHUNK_GZIP_SIZE = 1024 * 1024;
 
+function applyContextToError(error: string, moduleName: string): string {
```

Comment:
I think you flipped the params around here 😅

Reply:
Whoops, a little end-of-day distracted refactoring. Thanks for the catch :sweat_smile: